package.json

```diff
@@ -3,8 +3,9 @@
   "version": "0.0.1",
   "copyright": "Copyright © 2017-2018 Nebulas.io",
   "description": "Nebulas JavaScript PAY SDK",
-  "main": "index.js",
+  "main": "./dist/nebpay.js",
   "scripts": {
+    "npm-build": "parcel build ./nebpay.js",
```

Comment:
抱歉，我没懂。已经有 `build` 脚本（gulp 调用 Browserify）来处理文件打包了，为什么还会有这个 `npm-build` 脚本（用 Parcel）来打包？

这两者都会生成 `./dist/nebpay.js` 文件，那后者存在的意义是啥？

Reply:
I am not familiar with browserify for module package, but the result of browserify packaging I tested before can not be used in other packaging tools (webpack), it is probably not meet UMD specification. You can also present your pull request, if you can generate a package that conforms to the UMD specification based on browserify (Of course, this is better. After all, browserify is already used in the code).

Reply:
我英文一般般，还是写中文吧。 😂 

Webpack 自己就是打包工具，原生支持 CommonJS 的模块规范，所以不需要把 Browserify 打包后的结果（dist file）给它，直接把包的入口文件（main file）给它就行了。

Reply:
It's not good that the source code entry(index.js...) as a dependency of other projects, if you do that, the source code is relatively  inconvenient to use,  such as the need to install the project's dependencies for other projects(without npm package management), etc. And it will not work if you do not configure the same development dependencies(for example, I probably did not install "babel-preset-stage-0" that is what the source code needs to work).

Reply:
谢谢回复。我大致明白你的意思了。